README.md

```diff
@@ -26,7 +26,7 @@
 <!-- /TOC -->
 ## Prerequisite
 
-To get some hands-on experience on the Fluent Operator, you'll need a Kind cluster. You also need to set up a Kafka cluster and an Elasticsearch cluster in this Kind cluster.
+To get some hands-on experience on the Fluent Operator, you'll need a minikube cluster. You also need to set up a Kafka cluster and an Elasticsearch cluster in this Kind cluster.
 
 ```shell
 # Setup a minikube cluster
```

Comment:
We need to add a note here saying: run `./create-minikube-cluster-for-mac.sh` to install minikube on a MacBook